### DISCUSSION TITLE
Instructions

### DISCUSSION THREAD

**Discussion by Raezerst** [Almamu/EVESharp#23] [Q&A]
Something you may want to add to the import instructions.  The default max_prepared_stmt_count for Mariadb 10.5 is like 16580. That isn’t enough.  I believe the command I used was.  Set global max_prepared_stmt_count=50000;
Maybe overkill but it worked.

**Comment by Almamu** [comment]
That was introduced due to the update of C# connector for MySQL and I haven't had time to properly document what changes there are needed. Hopefully should be solved in the following week as I finish the rewrite of the proxy-node-proxy architecture used.
